### PR TITLE
nwg-clipman: init at 0.2.3

### LINKED ISSUE #326289: Package request: nwg-clipman
**Project description**

Clipboard manager for Wayland; part of the nwg-shell project

**Metadata**

* homepage URL: https://github.com/nwg-piotr/nwg-clipman
* source URL: https://github.com/nwg-piotr/nwg-clipman
* license: mit
* platforms: linux

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by AveryanAlex**
## Init nwg-clipman at 0.2.3

Resolves #326289

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by doronbehar** [comment]
CI error.

**Comment by averyanalex** [comment]
> CI error.

I added myself as maintainer in unmerged #326213

File: pkgs/by-name/nw/nwg-clipman/package.nix [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+, cliphist
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "nwg-clipman";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "nwg-piotr";
+    repo = pname;
```
Comment (Aleksanaa) [inline pkgs/by-name/nw/nwg-clipman/package.nix:18]:
```suggestion
    repo = "nwg-clipman";
```

File: pkgs/by-name/nw/nwg-clipman/package.nix [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  nativeCheckInputs = [
+    wl-clipboard
+    cliphist
+  ];
+
+  postInstall = ''
+    mkdir -p $out/share/{applications,pixmaps}
+    cp $src/nwg-clipman.desktop $out/share/applications/
+    cp $src/nwg-clipman.svg $out/share/pixmaps/
```
Comment (Aleksanaa) [inline pkgs/by-name/nw/nwg-clipman/package.nix:55]:
```suggestion
    install -Dm644 nwg-clipman.desktop -t $out/share/applications/
    install -Dm644 nwg-clipman.svg -t $out/share/pixmaps/
```

**Comment by Aleksanaa** [comment]
@ofborg eval

**Comment by Aleksanaa** [comment]
@ofborg eval

**Comment by Aleksanaa** [comment]
@ofborg build nwg-clipman

**Review by Aleksanaa — APPROVED** [review]

**Pull request merged**